data/common.yaml

```diff
@@ -14,12 +14,10 @@ mcollective::server_config:
   rpcauthorization: 1
   rpcauthprovider: "action_policy"
   rpcaudit: 1
-  rpcauditprovider: "choria"
-  plugin.rpcaudit.logfile: "/var/log/puppetlabs/mcollective-audit.log"
   factsource: "yaml"
-  plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
-  logfile: "/var/log/puppetlabs/mcollective.log"
-  daemonize: 1
+  plugin.yaml: "/etc/choria/generated-facts.yaml"
+  logfile: "/var/log/choria-mcorpc.log"
+  daemonize: 0
```

Comment:
This looks suspect from a commit that relocate files :thinking:

Reply:
Could even just remove it - its server config, now entirely unused. So just going on a big cleanup spree really